src/fe-text/statusbar-config.c

```diff
@@ -463,23 +522,38 @@ static CONFIG_NODE *statusbar_items_section(CONFIG_NODE *parent)
         return parent;
 }
 
-/* SYNTAX: STATUSBAR <name> ADD [-before | -after <item>]
-           [-priority #] [-alignment left|right] <item> */
-static void cmd_statusbar_add(const char *data, void *server,
-			      void *item, CONFIG_NODE *node)
+/* SYNTAX: STATUSBAR ADDITEM|MODIFYITEM [-before | -after <item>]
+           [-priority #] [-alignment left|right] <item> <statusbar> */
```

Comment:
Not sure about additem being `<item>` first `<statusbar>` last

Reply:
it somewhat matches `/channel add <channel> <network>`

Reply:
I guess. No strong opinion either way.